source/connect/connection-options.txt

```diff
@@ -0,0 +1,352 @@
+       | **Connection URI Example**: ``serverSelectionTimeoutMS=40000``
+
+   * - :manual:`serverSelectionTryOnce </reference/connection-string/#mongodb-urioption-urioption.serverselectiontryonce>`
+     - | **Data Type**: {+bool-data-type+}
+       | **MongoDB\\Client Example**: ``$uriOptions = [ 'serverSelectionTryOnce' => false ];``
+       | **Connection URI Example**: ``serverSelectionTryOnce=false``
+
+   * - :manual:`heartbeatFrequencyMS </reference/connection-string/#mongodb-urioption-urioption.heartbeatfrequencyms>`
+     - | **Data Type**: {+int-data-type+}
+       | **MongoDB\\Client Example**: ``$uriOptions = [ 'heartbeatFrequencyMS' => 50000 ];``
+       | **Connection URI Example**: ``heartbeatFrequencyMS=50000``
+
+   * - :manual:`socketCheckIntervalMS </reference/connection-string/#mongodb-urioption-urioption.socketcheckintervalms>`
+     - | **Data Type**: {+int-data-type+}
+       | **MongoDB\\Client Example**: ``$uriOptions = [ 'socketCheckIntervalMS' => 4000 ];``
+       | **Connection URI Example**: ``socketCheckIntervalMS=4000``
+
+Miscellaneous Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Connection Option
+     - Description
+   
+   * - :manual:`appName </reference/connection-string/#mongodb-urioption-urioption.appname>`
+     - | **Data Type**: {+string-data-type+}
+       | **MongoDB\\Client Example**: ``$uriOptions = [ 'appName' => 'myApp' ];``
+       | **Connection URI Example**: ``appName=myApp``
+
+   * - :manual:`retryReads </reference/connection-string/#mongodb-urioption-urioption.retryreads>`
+     - | **Data Type**: {+bool-data-type+}
+       | **MongoDB\\Client Example**: ``$uriOptions = [ 'retryReads' => false ];``
+       | **Connection URI Example**: ``retryReads=false``
+
+   * - :manual:`retryWrites </reference/connection-string/#mongodb-urioption-urioption.retrywrites>`
+     - | **Data Type**: {+bool-data-type+}
+       | **MongoDB\\Client Example**: ``$uriOptions = [ 'retryWrites' => false ];``
+       | **Connection URI Example**: ``retryWrites=false``
+   
+   * - :manual:`loadBalanced </reference/connection-string/#mongodb-urioption-urioption.loadbalanced>`
+     - | **Data Type**: {+bool-data-type+}
+       | **MongoDB\\Client Example**: ``$uriOptions = [ 'loadBalanced' => true ];``
+       | **Connection URI Example**: ``loadBalanced=false``
+
+   * - :manual:`srvMaxHosts </reference/connection-string/#mongodb-urioption-urioption.srvmaxhosts>`
+     - | **Data Type**: {+int-data-type+}
+       | **MongoDB\\Client Example**: ``$uriOptions = [ 'srvMaxHosts' => 5 ];``
+       | **Connection URI Example**: ``srvMaxHosts=5``
+
+API Documentation
+-----------------
+
+For more information about the types used on this page, see the following API
+documentation:
+
+- :ref:`MongoDB\Client <php-mongodbclient>`
+- `MongoDB\\Driver\\ReadPreference <https://www.php.net/manual/en/class.mongodb-driver-readpreference.php>`__
```

Comment:
Q: should we separate the PHP manual pages from the MongoDB-hosted API docs, or just group them together as "API documentation"? I separated them in [this PR](https://github.com/mongodb/docs-php-library/pull/110/files#diff-ac016ef66e1fc636f4229cf8150ad9db6ce31e2baa5a814de779ddbe5ad227a0R170-R180), for example, but open to either